uaclient/entitlements/base.py

```diff
@@ -69,6 +77,16 @@ def description(self) -> str:
         """A sentence describing this entitlement"""
         pass
 
+    @property
+    def presentation_name(self) -> str:
+        """The user-facing name shown for this entitlement"""
+        return (
+            self.cfg.entitlements.get(self.name, {})
+            .get("entitlement", {})
+            .get("affordances", {})
+            .get("presentAs", self.name)
```

Comment:
good it's clear and falls back to defined original entitlement.name in absence of presentAs.

Reply:
What happens if the user is `unattached` ? On Focal, we would return `cis` then ?